### PR TITLE
Add possibility to show image on events.

### PR DESCRIPTION
**Pull request by busykoala**
Part of: https://4teamwork.atlassian.net/browse/OGB-480

TODO:
- [x] Extend schema, view and template
- [x] Test new feature
- [x] Translate
- [x] Add changelog entry
- [x] Style

image

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -2,12 +2,13 @@ Changelog
 =========
 
 
-1.14.5 (unreleased)
+1.15.0 (unreleased)
```
Comment (Nachtalb) [inline docs/HISTORY.txt:5]: The release version is usually chosen on when releasing in the case until then new changes come that have an impact on the version and is automatically adjusted by `zest.releaser`.

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -2,12 +2,13 @@ Changelog
...
 -------------------
 
 - Fix german formatted date translation for open-ended events [Nachtalb]
 - Fix plone 5 upgrade step, which accidentally broke ftw.keywordwidget [mathias.leimgruber]
 - Remove obsolete dependency from Plone 5. From now on install the "plone4" extra for Plone 4 installations [Nachtalb]
+- Add possibility to show a leadimage within the block. [busykoala]
```
Comment (Nachtalb) [inline docs/HISTORY.txt:11]: s/block/event listing block/

File: docs/HISTORY.txt (hunk 1 above)
Comment (busykoala) [reply docs/HISTORY.txt:5]: I do that so I remember that it's necessary to go up by minor/major. If it's only a bugfix I'd leave it free as well.
It's just a reminder for myself...

File: docs/HISTORY.txt (hunk 2 above)
Comment (busykoala) [reply docs/HISTORY.txt:11]: Is there another one?

**Pull request merged**